woohoo.rb

```diff
@@ -0,0 +1,11 @@
+[42, 999].inject(&:+) # should complain
+[42, 999].reduce(&:+) # should not complain
+[42, 999].find { |_| true } # should not complain
+[42, 999].detect { |_| true } # should complain
+
+[123, 42, 999].inject(&:+) # should complain
+[123, 42, 999].reduce(&:+) # should not complain
+[123, 42, 999].find { |_| true } # should not complain
+[123, 42, 999].detect { |_| true } # should complain
+
+"HOUND BITE ME!"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.